packages/storage/package.json

```diff
@@ -19,8 +19,12 @@
     "dev": "rollup -c -w",
     "test": "run-p test:browser lint",
     "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:browser",
-    "test:browser:unit": "karma start --single-run --unit",
-    "test:browser:integration": "karma start --single-run --integration",
+    "test:browser:compat:unit": "karma start --single-run --compat --unit",
+    "test:browser:exp:unit": "karma start --single-run --exp --unit",
+    "test:browser:compat:integration": "karma start --single-run --compat --integration",
+    "test:browser:exp:integration": "karma start --single-run --exp --integration",
+    "test:browser:compat": "karma start --single-run --compat",
+    "test:browser:exp": "karma start --single-run --exp",
```

Comment:
Should we add these new test targets to the CI rule?

Reply:
These were for development testing convenience, CI tests everything.

Reply:
Please leave them. It makes everyone's life easier.